lib/rubocop/cop/style/array_intersect.rb

```diff
@@ -34,6 +39,15 @@ module Style
       #   array1.intersect?(array2)
       #   !array1.intersect?(array2)
       #
+      #   # bad
+      #   array1.intersection(array2).any?
+      #   array1.intersection(array2).empty?
+      #   array1.intersection(array2).none?
+      #
+      #   # good
+      #   array1.intersect?(array2)
+      #   !array1.intersect?(array2)
```

Comment:
`good` examples may simply be moved. i.e.,:

```
      # @example
      #   # bad
      #   (array1 & array2).any?
      #   (array1 & array2).empty?
      #   (array1 & array2).none?
      #
      #   # bad
      #   array1.intersection(array2).any?
      #   array1.intersection(array2).empty?
      #   array1.intersection(array2).none?
      #
      #   # good
      #   array1.intersect?(array2)
      #   !array1.intersect?(array2)
```

Reply:
Fair enough! Updated.